app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
```

Comment:
Go through the filename of this component and explicitly point out in a comment whether you find any issues with the naming or not. If you find an issue, please do point out what is the issue that you are seeing.

Reply:
I wasn't sure if I should alias the table imported from NeetoUI. Naming both the component and the import as Table creates conflict. I'll be sure to alias imports from now on.

Reply:
Aliasing imports is the right way to go.